platform/email/platform-email-impl/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should probably go into the documentation. See `distribution/docs/src/main/resources/_contents/_tables` for examples of the tables that map to the metatype configuration. 

Unless we put it somewhere else, that table should be included in the `managing-platform-contents.adoc` file